editor.planx.uk/src/routes/pay.tsx

```diff
@@ -0,0 +1,101 @@
+import gql from "graphql-tag";
+import { client } from "lib/graphql";
+import {
+  compose,
+  mount,
+  NaviRequest,
+  NotFoundError,
+  redirect,
+  route,
+  withData,
+  withView,
+} from "navi";
+import { useStore } from "pages/FlowEditor/lib/store";
+import InviteToPay from "pages/Pay/InviteToPay";
+import MakePayment from "pages/Pay/MakePayment";
+import { PaymentRequest } from "pages/Pay/types";
+import React from "react";
+
+import { getTeamFromDomain, makeTitle, validateTeamRoute } from "./utils";
+import standaloneView from "./views/standalone";
+
+const payRoutes = compose(
+  withData(async (req) => {
+    const externalDomainTeam = await getTeamFromDomain(
+      window.location.hostname
+    );
+
+    return {
+      mountpath: req.mountpath,
+      team: req.params.team || externalDomainTeam,
+      isPreviewOnlyDomain: Boolean(externalDomainTeam),
+    };
+  }),
+
+  withView(async (req) => {
+    await validateTeamRoute(req);
+    return await standaloneView(req);
+  }),
+
+  mount({
+    "/": route(async (req) => {
+      const paymentRequest = await getPaymentRequest(req);
+      return {
+        title: makeTitle("Make a payment"),
+        view: <MakePayment {...paymentRequest} />,
+      };
+    }),
+    "/invite": route(async (req) => {
+      const paymentRequest = await getPaymentRequest(req);
+      return {
+        title: makeTitle("Invite to pay"),
+        view: <InviteToPay {...paymentRequest} />,
+      };
+    }),
+    "/pages/:page": redirect(
+      (req) => `../../../preview/pages/${req.params.page}`
+    ),
+    "/invite/pages/:page": redirect(
+      (req) => `../../../../preview/pages/${req.params.page}`
+    ),
+  })
+);
+
+const getPaymentRequest = async (req: NaviRequest) => {
+  const paymentRequestId = req.params["payment-request-id"];
```

Comment:
nit / question: I was manually navigating to these URLs based on a database record and realised I was defaultting to `paymentRequestId=` rather than hyphens because that's how we format S&R magic link params (eg `sessionId=`) - do we care about consistency like that here? I'm not sure it really matters, just a very minor observation!

Reply:
It's worth being consistent here, I've updated this to camelCase 👍